packages/mobile/src/harmony-native/components/input/SelectablePill/types.ts

```diff
@@ -3,9 +3,13 @@ import type { PressableProps, ViewProps } from 'react-native'
 import type { Icon } from 'app/harmony-native/icons'
 
 export type SelectablePillProps = {
+  type: 'button' | 'checkbox' | 'radio'
```

Comment:
Just curious, do we actually use it as a radio anywhere?

Reply:
yeah in select artist page in new sign-up, also on mobile saved page. checkbox is in select-genres page